openbb_platform/extensions/platform_api/openbb_platform_api/utils/widgets.py

```diff
@@ -175,9 +193,10 @@ def build_json(openapi: Dict):
                 "data": {
                     "dataKey": "results",
                     "table": {
-                        "showAll": False,
+                        "showAll": True,
```

Comment:
Won't this result in get-only routers not displaying anything because they don't have schemas set?

Reply:
That would be very counterintuitive for "showAll" to not actually do what it says it does.  When `columnsDefs` are defined, setting to False hides the fields without definitions. This param seems to have no effect when no `columnsDefs` are supplied.

![Screenshot 2024-09-18 at 8 29 11 AM](https://github.com/user-attachments/assets/59604a6b-b36b-497f-a689-c3270b074dc3)